match/lib/match/nuke.rb

```diff
@@ -148,9 +148,7 @@ def prepare_list
       # Get all iOS and macOS profile
       self.profiles = []
       prov_types.each do |prov_type|
-        types = profile_types(prov_type)
-        # Filtering on 'profileType' seems to be undocumented as of 2020-07-30
-        # but works on both web session and official API
+        types = Match.profile_types(prov_type)
```

Comment:
Same here. this information is actually mentioned in documentation at this moment (2022-05-22)
https://developer.apple.com/documentation/appstoreconnectapi/list_and_download_profiles#query-parameters